test/mailers/reservation_mailer_test.rb

```diff
@@ -0,0 +1,65 @@
+require "test_helper"
+
+class ReservationMailerTest < ActionMailer::TestCase
+  setup do
+    ActionMailer::Base.deliveries.clear
+  end
+
+  test "#reservation_requested includes HTML and text parts when a reservation is submitted" do
+    reservation = create(:reservation)
+
+    ReservationMailer.with(reservation:).reservation_requested.deliver_now
+
+    mail = ActionMailer::Base.deliveries.last
+
+    subject = "Reservation submitted for #{reservation.started_at.to_fs(:short_date)}"
+
+    assert mail.multipart?, "mail should be multipart"
```

Comment:
There's an [`assert_delivered_mail`](https://github.com/chicago-tool-library/circulate/blob/c94929772bd3ad971f2ce55edca6db09dfc2a88b/test/application_system_test_case.rb#L62)  utility function ([used like this](https://github.com/chicago-tool-library/circulate/blob/c94929772bd3ad971f2ce55edca6db09dfc2a88b/test/system/admin/gift_memberships_test.rb#L37)) currently used in system tests that might make this more concise.